src/app/player/player.component.ts

```diff
@@ -1,6 +1,6 @@
 import {Component, Input, OnChanges, SimpleChange, OnInit} from 'angular2/core';
 import {AsyncPipe} from 'angular2/common';
-import {DovetailPlayer} from '../../lib/dovetail-player';
+import {DovetailAudio} from '../../lib/dovetail_audio';
```

Comment:
I think DovetailAudio makes more sense as a wrapper around `Audio` even though it's a little clunkier. I think that `window.Audio = window.DovetailAudio` just makes more sense (though I need to shadow Audio before we can do that)